source/manual/local-frontend-development.html.md

```diff
@@ -5,111 +5,97 @@ layout: manual_layout
+## Using startup scripts
+If you are making changes to a frontend app and nothing else, you can view these changes by running the application `./startup.sh` script:
 
 ```shell
 cd /var/govuk/frontend
 ./startup.sh --live
 ```
 
-### Running a frontend app against changes in `govuk_publishing_components`
-
-Update the Gemfile for the frontend app to point to your local `govuk_publishing_components` gem:
+If you want to test changes in [govuk_publishing_components] against a frontend app, you need to edit your frontend app Gemfile and then run the startup script:
 
 ```ruby
 gem 'govuk_publishing_components', path: '../govuk_publishing_components'
 ```
 
-Then run your frontend app again pointing to live:
-
 ```shell
+bundle install
 ./startup.sh --live
 ```
 
-### Running a frontend app against changes in `static`
+## Using govuk-docker
 
-Make changes to the Gemfile of the frontend app you're working on as per [changes to govuk_publishing_components above](#running-a-frontend-app-against-changes-in-govuk_publishing_components), then set in `development.rb`:
+This assumes that you have already installed and setup [govuk-docker]. We will use [government-frontend] as an example here, but these instructions apply to any frontend app.
 
-```ruby
-...
-config.assets.debug = false
-...
+If you are making changes to a frontend app and nothing else, you can view these changes by running the following:
+
+```shell
+cd /var/govuk-docker
+make government-frontend
+
+cd /var/govuk/government-frontend
+govuk-docker up government-frontend-app-live
 ```
 
-If you're making changes to the [govuk_publishing_components] gem that affect [static] and are likely things across the whole site, e.g: cookie banner (and survey banner possibly), update the Gemfile of [static] to point to the gem:
+If you want to test changes in [govuk_publishing_components] against a frontend app, you need to edit your frontend app Gemfile first:
 
 ```ruby
-...
 gem 'govuk_publishing_components', path: '../govuk_publishing_components'
-...
 ```
 
-Run up static:
-
 ```shell
-cd /var/govuk/static
-govuk_setenv static ./startup.sh --live
+cd /var/govuk/government-frontend
+bundle install
+govuk-docker up government-frontend-app-live
 ```
 
-Run up frontend against local static (the `--live` flag means the rest runs against live data):
+If you want to test changes in static against a frontend app, you need to tell Docker to look at your local version of static rather than live:
 
 ```shell
-PLEK_SERVICE_STATIC_URI=localhost:3013 ./startup.sh --live
+cd /var/govuk/govuk-docker/projects/government-frontend
+vim docker-compose.yml
 ```
+Edit the docker-compose.yml live config to depend on static and remove the live static environment:
+
+```yaml
+  government-frontend-app-live:
+    <<: *government-frontend-app
+  depends_on:
+    ...
+    - static-app
+  environment:
+    ...
+    #PLEK_SERVICE_STATIC_URI: assets.publishing.service.gov.uk
+```
```

Comment:
Could this be more explicit? I didn't spot both changes required initially. Maybe..

`- static-app # <- add this line`

`#PLEK_SERVICE_STATIC_URI: assets.publishing.service.gov.uk <- comment out this line`